demo/app/ios/Runner/flutterPlugin.swift

```diff
@@ -67,6 +67,9 @@ public class SwiftWalletSDKPlugin: NSObject, FlutterPlugin {
 
         case "activityLogger":
             storeActivityLogger(result:result)
+        
+        case "getVersionDetail":
```

Comment:
maybe rename it to getWalletSDKVersion or something like this, to not confuse it with the app version

Reply:
yea make sense will do that